pyserini/search/pysearch.py

```diff
@@ -42,6 +42,10 @@ def __init__(self, document):
     def docid(self: JDocument) -> str:
         return self.object.getField('id').stringValue()
 
+    def id(self: JDocument) -> str:
```

Comment:
I don't think it is common to tell explicitly the type of self (`JDocument`), but that is ok